app/src/main/java/com/Alkemy/alkemybankbase/ui/activities/SignUpActivity.kt

```diff
@@ -48,6 +51,10 @@ class SignUpActivity : AppCompatActivity() {
             //show password error
             binding.etPassword.error = getString(resId)
         }
+        viewModel.isLoading.observe(this) {
+            if(it) showLoading()
+            else stopLoading()
+        }
```

Comment:
Se podría simplementa settear la visibilidad del progress bar acá y eliminar las funciones showLoading y stopLoading (en otras activities también).
binding.prgbar.visibility = if (it) View.VISIBLE else View.GONE